source/setup.py

```diff
@@ -219,10 +219,9 @@ def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
 		],
 		"includes": [
 			"nvdaBuiltin",
-			# #3368: bisect was implicitly included with Python 2.7.3, but isn't with 2.7.5.
-			"bisect",

```

Comment:
Wouldn't it be safer to keep this forcibly included? This PR has been written exactly because implicit includes turned out to be unreliable, and may break with the Python upgrades for add-ons. We have neither automatic tests to make sure this won't happen again for `bisect`, nor can guarantee that add-ons which rely on `bisect` would be tested in Alpha with the next Python upgrade before the release.